teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name: 'Unknown')
+    super(age, name: name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Please kindly fix the parameters used for the super() as explained in the review comment above
- [ ] Also kindly add the parent_permission to with default parameters to the initialize constructor :+1: